### ISSUE TITLE
gatsby init - invalid selection in "additional plugins" section

### ISSUE THREAD

**Issue by Takeno** [gatsbyjs/gatsby#32826]
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/gatsbyjs/gatsby/issues
- [X] This issue is not a question, feature request, RFC, or anything other than a bug report directly related to Gatsby. Please post those things in GitHub Discussions: https://github.com/gatsbyjs/gatsby/discussions

### Description

During `npm init gatsby`, in the section "Would you like to install additional features with other plugins?" user is able to press the spacebar to select plugins but if the spacebar is used on "done" element it is chosed as plugin to be installed. The result is `TypeError: Cannot read property 'plugins' of undefined`.

```
$ npm init gatsby

What would you like to call your site?
✔ · test
What would you like to name the folder where your site will be created?
✔ tmp/ test
✔ Will you be using a CMS?
· No (or I'll add it later)
✔ Would you like to install a styling system?
· No (or I'll add it later)
✔ Would you like to install additional features with other plugins?
· Done
TypeError: Cannot read property 'plugins' of undefined
    at /Users/xxx/.npm/_npx/7231/lib/node_modules/create-gatsby/lib/index.js:1:202670
    at Array.map (<anonymous>)
    at exports.run (/Users/xxx/.npm/_npx/7231/lib/node_modules/create-gatsby/lib/index.js:1:202648)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
```

### Reproduction Link

none

### Steps to Reproduce

1. npm init gatsby
2. type "test"
3. type "test"
4. Select NO cms
5. Select NO styling system
6. Go to "Done" and press spacebar
7. Press "Enter" to go ahead
...

### Expected Result

It should create the new project

### Actual Result

It crashes

```
TypeError: Cannot read property 'plugins' of undefined
    at /Users/xxx/.npm/_npx/7231/lib/node_modules/create-gatsby/lib/index.js:1:202670
    at Array.map (<anonymous>)
    at exports.run (/Users/xxx/.npm/_npx/7231/lib/node_modules/create-gatsby/lib/index.js:1:202648)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
```

### Environment

```shell
System:
    OS: macOS 11.4
    CPU: (12) x64 Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 14.17.3 - ~/.nvm/versions/node/v14.17.3/bin/node
    Yarn: 1.22.10 - ~/.nvm/versions/node/v14.17.3/bin/yarn
    npm: 6.14.13 - ~/.nvm/versions/node/v14.17.3/bin/npm
  Languages:
    Python: 2.7.16 - /usr/bin/python
  Browsers:
    Chrome: 92.0.4515.131
    Firefox: 91.0
    Safari: 14.1.1
```

**Comment by LekoArts** [comment]
Hi, thanks for the issue!

Seems like our current wording on this step is confusing:

> **? Would you like to install additional features with other plugins?**
(Multiple choice) Use arrow keys to move, enter to select, and choose "Done" to confirm your choices

What you really should do:

- Use spacebar to select items
- Move cursor to "Done" and press "Enter" to confirm

Then it'll work :) I'll put up a PR changing the wording!

**Comment by Takeno** [comment]
I was creating a new project and I used the spacebar as default action :) 

I think the best solution is to avoid Done is selectable OR ignore it if selected, but I don't know how it is implemented. So fixing the wording is a quick solution for sure.

> What you really should do:
>
>   -  Use spacebar to select items
>    - Move cursor to "Done" and press "Enter" to confirm

Just to be sure, the Enter button still works to select items. I also tried all the init flow using only the enter button instead of spacebar and it worked. 

I hope this consideration can help your work.
Thanks!
